tests/test_pipelines_common.py

```diff
@@ -242,3 +248,67 @@ def _test_pipeline(self, nlp: Pipeline):
                 self.assertIn(key, result)
 
         self.assertRaises(Exception, nlp, self.invalid_inputs)
+
+
+class DummyTok:
```

Comment:
This class uses `torch` so it should probably be inside an `if is_torch_available()` block.

Reply:
I will ignore for the time being. I think this might change anyway linked to this discussion: https://github.com/huggingface/transformers/pull/9432#discussion_r552546793